extension/src/popup/components/manageAssets/TrustlineError/index.tsx

```diff
@@ -143,15 +144,19 @@ export const TrustlineError = ({
     : TRUSTLINE_ERROR_STATES.UNKNOWN_ERROR;
 
   return (
-    <div className="TrustlineError">
-      <div className="TrustlineError__body">
-        <RenderedError
-          errorState={errorState}
-          assetBalance={assetBalance}
-          resultCodes={JSON.stringify(getResultCodes(error))}
-        />
-      </div>
-      <div className="TrustlineError__button">
+    <View>
```

Comment:
without this - the extension had no width and was "collapsing" into a long skinny screen

Reply:
Thanks for fixing this! I noticed it when writing tests for trustlines.